### PR TITLE
[office-js-preview] (Excel) Pulling latest Excel type definitions from CDN

### PR DESCRIPTION
**Pull request by alison-mk**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: Provide a URL to documentation or source code which provides context for the suggested changes:
Beta URL: https://appsforoffice.microsoft.com/lib/beta/hosted/office.debug.js
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/office-js-preview/index.d.ts [hunk 1]
```diff
@@ -38219,6 +38219,22 @@ declare namespace Excel {
...
+        /**
+         * Gets the data source of the whole chart.
+         *
+         * @remarks
+         * [Api set: ExcelApi BETA (PREVIEW ONLY)]
+         * @beta
+         */
+         getDataRange(): OfficeExtension.ClientResult<string>;
+         /**
+          * Gets the data source of the whole chart. If the current data source is not supported yet, this method will not throw exception.
```
Comment (alison-mk) [inline types/office-js-preview/index.d.ts:38231]: Note to reviewers: I'm doing internal research to clarify this description.

Comment (Rick-Kirkham) [inline types/office-js-preview/index.d.ts:38231]: I believe the descriptions for getDataRange and getDataRangeOrNullObject are reversed. The pattern for our get* and get*OrNullObject methods is that the former throws an exception if the entity isn't there and the latter does NOT throw an exception. Instead, it returns a proxy object whose isNullObject property is set to true.

**Review by Rick-Kirkham — CHANGES_REQUESTED** [review]

File: types/office-js-preview/index.d.ts (hunk 1 above)
Comment (alison-mk) [reply types/office-js-preview/index.d.ts:38231]: Thanks, Ricky! This is very helpful. I will confirm with PM/eng.

Comment (alison-mk) [reply types/office-js-preview/index.d.ts:38231]: I updated this based on your feedback + internal discussion. Please take a look when you have a chance. Thanks!

**Review by Rick-Kirkham — APPROVED** [review]

**Comment by alison-mk** [comment]
Ready to merge

**Pull request merged**